README.md

```diff
@@ -219,6 +219,10 @@ optional arguments:
                         Upgrades information table
   -d DIR [DIR ...], --dir DIR [DIR ...]
                         Delta directories (space-separated)
+  -u VERSION, --max-version VERSION
+                        Upper bound limit version to run the deltas up to.
+  -v VARIABLE VALUE, --var VARIABLE VALUE
+                        Assign variable for running SQL deltas.
```

Comment:
I don't find this description easy to understand for users. And it is not described how to use variables in SQL delta files.

Reply:
I now realize that there's https://github.com/opengisch/pum/pull/42  that improves the doc situation.